data/rule_data.yml

```diff
@@ -180,54 +180,60 @@ rule_data:
   # No releases on Fridays and weekends
   # https://enterprisecontract.dev/docs/ec-policies/release_policy.html#schedule__weekday_restriction
   disallowed_weekdays:
-  - friday
-  - saturday
-  - sunday
+    - friday
+    - saturday
+    - sunday
 
   # No releases during year-end shutdown
   # https://enterprisecontract.dev/docs/ec-policies/release_policy.html#schedule__date_restriction
   disallowed_dates:
-  # EOY 2024
-  - 2024-12-24
-  - 2024-12-25
-  - 2024-12-26
-  - 2024-12-27
-  - 2024-12-28
-  - 2024-12-29
-  - 2024-12-30
-  - 2024-12-31
-  - 2025-01-01
-  # EOY 2025
-  - 2025-12-24
-  - 2025-12-25
-  - 2025-12-26
-  - 2025-12-27
-  - 2025-12-28
-  - 2025-12-29
-  - 2025-12-30
-  - 2025-12-31
-  - 2026-01-01
-
-# Usage: https://enterprisecontract.dev/docs/ec-policies/release_policy.html#olm__required_olm_features_annotations_provided
+    # Thanksgiving 2024
+    - 2024-11-27
+    - 2024-11-28
+    - 2024-11-29
+    - 2024-11-30
```

Comment:
I don't think you need to include Nov 30th and Dec 1st since those are on the weekend. Probably no harm in including them.